CMakeLists.txt

```diff
@@ -54,13 +54,13 @@ endif()
 
 if(UNIX AND NOT APPLE)
   message("Building for Linux")
-  add_definitions( -D_DEBUG -D__LINUX__ -DLINUX -D_MAX_PATH=260 -D_MAX_FNAME=256 -D_REENRANT -D__32BIT__ -DHAVEALLOCA_H -D_USE_OGL_ACTIVE_TEXTURES)
+  add_definitions( -D_DEBUG -D__LINUX__ -DLINUX -D_MAX_PATH=260 -D_MAX_FNAME=256 -D_REENRANT -D__32BIT__ -DHAVEALLOCA_H -D_USE_OGL_ACTIVE_TEXTURES -DLOGGER)
   set(PLATFORM_INCLUDES "lib/linux" ${SDL_INCLUDE_DIR})
 endif()
 
 if(APPLE)
   message("Building for MAC OSX")
-  add_definitions(-D_DEBUG -D__LINUX__ -DLINUX -D_MAX_PATH=260 -D_MAX_FNAME=256 -D_REENRANT -DMACOSX=1 -D_USE_OGL_ACTIVE_TEXTURES)
+  add_definitions(-D_DEBUG -D__LINUX__ -DLINUX -D_MAX_PATH=260 -D_MAX_FNAME=256 -D_REENRANT -DMACOSX=1 -D_USE_OGL_ACTIVE_TEXTURES -DLOGGER)
```

Comment:
we want this definition conditionally, if the user specified the CMake [Option](https://cmake.org/cmake/help/latest/command/option.html) LOGGER

Reply:
Added in latest commit